tests/Unit/Http/SessionServiceProviderTest.php

```diff
@@ -53,13 +53,13 @@ public function testRegister()
                 [Session::class],
                 [
                     NativeSessionStorage::class,
-                    ['options' => ['cookie_httponly' => true, 'name' => 'session'], 'handler' => null],
+                    ['options' => ['cookie_httponly' => true, 'name' => 'session', 'lifetime' => 30], 'handler' => null],
```

Comment:
FILE: /var/www/tests/Unit/Http/SessionServiceProviderTest.php
----------------------------------------------------------------------
FOUND 0 ERRORS AND 2 WARNINGS AFFECTING 2 LINES
----------------------------------------------------------------------
 56 | WARNING | Line exceeds 120 characters; contains 121 characters
 62 | WARNING | Line exceeds 120 characters; contains 132 characters
----------------------------------------------------------------------

Reply:
done